packages/salesforcedx-vscode-soql/package.json

```diff
@@ -19,28 +19,45 @@
     "Programming Languages"
   ],
   "dependencies": {
+    "@salesforce/sobject-metadata": "0.1.4",
     "@salesforce/soql-builder-ui": "0.0.2",
     "@salesforce/soql-language-server": "0.2.1",
     "debounce": "^1.2.0",
+    "vscode-extension-telemetry": "0.1.6",
     "vscode-languageclient": "^6.1.3"
   },
   "devDependencies": {
+    "@salesforce/salesforcedx-test-utils-vscode": "*",
```

Comment:
@lcampos - our PRs broke because we didn't pick up the bump change to 49.9.0 that happened in develop.  Would it be helpful to just (star) these and control the version elsewhere in a single place, like in salesforcedx-vscode-core?

Reply:
No, just rebase the PR to the latest changes in develop. We need to keep this around or lerna won't be able to create the symlinks in development or run the build tasks in circleci